operators/transformation/mergemap.md

```diff
@@ -12,6 +12,20 @@
 
 ---
 
+### Why use `mergeMap`?
+
+`mergeMap` is for you when you want to observe all action from the source's emission
+after you've mapped them to a new observable. In other word, you want to see what
+each item is doing once you've passed them through your function.
+
+These emission could behave differently depending on the function you provided and 
+herein lies the strength of `mergeMap`.  This operators subcribes to all mapped 
+emissions, giving you new output as they come.  It doesn't wait for anything to
+complete or unsubscribes from anything else.  You simply get an unfiltered stream
+of output.  Keep in mind this is a double edge sword.  If the source or inner
```

Comment:
I would reword this a bit, if the inner observables are long lived and are not properly cleaned up then you could run into a memory leak. If this is the case, and you still want to use mergeMap you may want to utilize takeUntil or another limiting operator to clean up the subscription at the correct time.

Reply:
Probably should mention concurrency here.